StreamVideoTests/Mock/MockRTCPeerConnection.swift

```diff
@@ -71,7 +71,11 @@ final class MockRTCPeerConnection: StreamRTCPeerConnectionProtocol, Mockable {
 
     // MARK: - Implementation
 
-    var remoteDescription: RTCSessionDescription?
+    var remoteDescription: RTCSessionDescription? {
+        get { self[dynamicMember: \.remoteDescription] }
```

Comment:
why do we need this?

Reply:
Just to be aligned with the Mockable protocol. It's not necessary but it can be problematic if we don't do it like this as stubbing the property via Mockable won't have an effect on the actual property.